spec/system/support_interface/ucas_matches_spec.rb

```diff
@@ -133,6 +137,9 @@ def when_i_follow_the_link_to_ucas_match_for_a_candidate
   end
 
   def then_i_should_see_ucas_match_summary
+    expect(page).to have_content 'Trackable applicant key'
+    expect(page).to have_content '0F8FB8240C73AB94'
```

Comment:
The same content appears in the match tables below anyway — consider a `within` block?